modules/ROOT/pages/afm-monitoring-public-apis.adoc

```diff
@@ -0,0 +1,53 @@
+
+* *Second relationship type (represented by dashed green arrows):*
++
+Monitor 2 runs on two separate schedules, and each is associated with the same public location. Monitor 2 is written to test one or more endpoints in public API 3 and in public API 4, though it could also test endpoints in more public APIs. The two APIs are in a single environment.
+
+* *Third relationship type (represented by dashed-and-dotted blue arrows):*
++
+Monitor 3 runs on two separate schedules, and each is associated with a different public location. Monitor 3 is written to test one or more endpoints in public API 5 and in public API 6, though it could also test endpoints in more public APIs. The two APIs are in a single environment.
+
+* *Fourth relationship type (represented by dotted orange arrows):*
++
+Monitor 4 runs on two separate schedules, and each is associated with the same public location, though each could be associated with a different public location. Each of the two tested public APIs is in a different environment. One environment could be a test environment, and the other environment could be a production environment. Public API 7 and public API 8 are copies of the same API. Monitor 4 is written to test one or more endpoints in public API 7 and the same endpoints in public API 8, though it could also test endpoints in more public APIs.
+
+* *Fifth relationship type (represented by dashed, dark blue lines):*
++
+Monitor 5 runs on a single schedule, though it could run on two. The schedule is associated with Public Location 2. Monitor 5 is written to test one or more endpoints in public API 9 and in public API 10 (though it could also test endpoints in more public APIs), both of which are accessible on the World-Wide Web.
+
+== Length of Time Before Tests Time Out
+
+To prevent tests run in public locations from consuming too many resources, AFM prevents tests from waiting longer than 120 seconds for a response from an endpoint. You cannot increase this wait time.
+
+== Scheduling Tests
+
+Tests are run at fixed intervals that you define in schedules. Each test is associated with a schedule. When you run tests against the endpoints of public APIs, AFM sets limitations on the following aspects of scheduling:
+
+Number of Schedules::
++
+Initially, AFM lets your organization in Anypoint Platform run up to five workers across public locations at one time. Therefore, you can run up to five tests of the endpoints in public APIs at one time.
++
+The number of workers that are running tests at any particular time is determined by the schedules that you associate with tests. Across the monitors that are created in AFM in your organization, there can be only five schedules. If you reach the limit and want to run more tests simultaneously, contact your MuleSoft administrator about raising your maximum.
```

Comment:
The number of workers is always the same (this type of worker/location is provisioned by us, thus users cannot create public locations) 
Also I would add that you can create private location if you reach the limit.

Reply:
Resolved.